cpp/src/parquet/parquet_version.h.in

```diff
@@ -18,14 +18,16 @@
 #ifndef PARQUET_VERSION_H
 #define PARQUET_VERSION_H
 
-#define PARQUET_VERSION_MAJOR @PARQUET_VERSION_MAJOR@
-#define PARQUET_VERSION_MINOR @PARQUET_VERSION_MINOR@
-#define PARQUET_VERSION_PATCH @PARQUET_VERSION_PATCH@
+#include "arrow/util/config.h"
+
+#define PARQUET_VERSION_MAJOR @ARROW_VERSION_MAJOR@
+#define PARQUET_VERSION_MINOR @ARROW_VERSION_MINOR@
+#define PARQUET_VERSION_PATCH @ARROW_VERSION_PATCH@
 
 #define PARQUET_SO_VERSION "@ARROW_SO_VERSION@"
 #define PARQUET_FULL_SO_VERSION "@ARROW_FULL_SO_VERSION@"
 
 // define the parquet created by version
-#define CREATED_BY_VERSION "parquet-cpp version @PARQUET_VERSION@"
+#define CREATED_BY_VERSION "parquet-cpp-arrow version @ARROW_VERSION@"
```

Comment:
Is it useful to also add the git commit SHA? I can't tell if it gets added, from looking at the change

Reply:
It isn't added here or in arrow.  Releases are tied to a git sha

Reply:
not to say including a the sha hash isn't a good idea but I think it should be outside the scope of this work.

Reply:
I understand, this change is already great, given how long the version hasn't changed